docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -284,10 +284,10 @@ Where *closedBoxId* is the id of the closed box and *boxKey* is correct proof fo
 Custom Proof / Proposition creation
 ###################################
 
-A proposition is a locker for a box, and Proof is an unlocker for a box. For some reason, a way how the box is locked/unlocked can be changed by the SDK developer. For example, a special box can be opened by two or more independent private keys. For such reason, custom Proof / Proposition can be created.
+A proposition is a locker for a box, and Proof is an unlocker for a box. For some reason, the manner in which the box is locked/unlocked can be changed by the SDK developer (should this be sidechain developer?). For example, a special box can be opened by two or more independent private keys. For such reason, custom Proof/Proposition can be created.
```

Comment:
@zennermac - A proposition is a locker for a box, and a proof is an unlocker for a box. The manner in which a box is locked/unlocked can be changed by the developer. For example, a special box can be opened by two or more independent private keys. For this reason, a custom Proof/Proposition can be created.